src/pydata_sphinx_theme/assets/styles/_navbar.scss

```diff
@@ -51,26 +53,73 @@
 // If we want the shadow to only point downward in the future, set
 // box-shadow to: 0 0.125rem 0.25rem -0.125rem rgba(0, 0, 0, 0.11);
 .navbar-light {
-  background: #fff !important; // Overrides bootstrap
+  background: var(
+    --pst-color-navbar-background
+  ) !important; // Overrides bootstrap
   box-shadow: 0 0.125rem 0.25rem 0 rgba(0, 0, 0, 0.11);
 
   .navbar-nav {
     li a.nav-link {
       padding: 0 0.5rem;
-      color: rgba(var(--pst-color-navbar-link), 1);
+      color: var(--pst-color-navbar-link);
 
       &:hover {
-        color: rgba(var(--pst-color-navbar-link-hover), 1);
+        color: var(--pst-color-navbar-link-hover);
+      }
+
+      &:focus {
+        color: var(--pst-color-navbar-link-hover);
       }
     }
 
     > .active > .nav-link {
       font-weight: 600;
-      color: rgba(var(--pst-color-navbar-link-active), 1);
+      color: var(--pst-color-navbar-link-active);
     }
   }
 }
 
 .navbar-header a {
   padding: 0 15px;
 }
+
+// inline the element in the navbar as long as they fit and use display block when colapsing
+@media (min-width: 768px) {
+  .navbar-center-item {
+    display: inline-block;
+  }
+}
+
+// the icons for theme change
+#theme-switch {
+  border-color: var(--pst-color-navbar-link-hover);
```

Comment:
```suggestion
  border-color: var(--pst-color-navbar-link-hover);
  margin-right: 0.4rem;
```

@12rambau, WDYT about this little change? Modifying the css on the flight looks better, IMHO.

with this change...
<img width="281" alt="Screen Shot 2022-04-07 at 12 20 46" src="https://user-images.githubusercontent.com/1640669/162234469-b7dd4ffd-b550-4fc0-b1ce-b9ea430e8824.png">
vs
<img width="269" alt="Screen Shot 2022-04-07 at 12 22 22" src="https://user-images.githubusercontent.com/1640669/162234677-550be9ad-ca04-4710-99e1-68f16d13484c.png">

Not a blocker for merging, just a nice to have, if you agree...

Reply:
Btw, I plan to merge this PR in the next 24-48 hs.